src/main/java/com/maxmind/geoip2/model/DomainResponse.java

```diff
@@ -19,6 +19,13 @@ public class DomainResponse extends AbstractResponse {
     private final String ipAddress;
     private final Network network;
 
+    /**
+     * Constructs an instance of {@code DomainResponse}.
+     *
+     * @param domain    the second level domain associated with the IP address
+     * @param ipAddress the IP address that the data in the model is for
+     * @param network   the network associated with the record
+     */
```

Comment:
It looks like the next `DomainResponse` constructor does not have any docs. I think we must be missing a rule to enforce these.

Reply:
This was also caused by <property name="minLineCount" value="2"/>